include/bringauto/logging/FileSink.hpp

```diff
@@ -64,6 +64,9 @@ class FileSink: public Sink {
 			auto ownerId = buf.st_uid;
 			auto groupID = buf.st_gid;
 			auto myId = getuid();
+			if(myId == 0) { // is running with root privileges
+				return true;
+			}
 
```

Comment:
Add else statement. This looks a bit like random returning in the middle of function.

Reply:
Plus on line 35 use the new SizeLiteral

Reply:
Line 35 repaired + I will not add else statement, because it would be too much intented